api/src/integration/java/org/pmiops/workbench/google/DirectoryServiceImplIntegrationTest.java

```diff
@@ -53,7 +53,14 @@ public void testCreateAndDeleteTestUser() {
                 });
     // Ensure our two custom schema fields are correctly set & re-fetched from GSuite.
     assertThat(aouMeta).containsEntry("Institution", "All of Us Research Workbench");
-    assertThat(service.getContactEmailAddress(userName)).isEqualTo("notasecret@gmail.com");
+    assertThat(service.getContactEmail(userName).get()).isEqualTo("notasecret@gmail.com");
```

Comment:
nit: If you do
```
import static com.google.common.truth.Truth8.assertThat;
```
in addition to the main `assertThat()`, you can use 
```
assertThat(service.getContactEmail(userName)).hasValue("notasecret@gmail.com");
```

Reply:
Good call, done. (Is there a good way to get IntelliJ to offer to auto-import the Truth8 version when I want to use this pattern?)